app/_src/production/dp-config/dpp-on-universal.md

```diff
@@ -9,7 +9,19 @@ When transparent proxying is not enabled, the outbound service dependencies have
 This also means that without transparent proxying **you must update** your codebases to consume those external services on `127.0.0.1` on the port specified in the `outbound` section.
 
 {% tip %}
-To avoid users bypassing the sidecar, have the service listen only on the internal interface (`127.0.0.1` or `::1`) instead of all interfaces (`0.0.0.0` or `::`).
+To avoid users bypassing the sidecar, have the service listen only on the internal interface (`127.0.0.1` or `::1`) instead of all interfaces (`0.0.0.0` or `::`). Another approach is to set the `serviceAddress` as shown below.
```

Comment:
can we explain what it does?

Reply:
I've changed the location and added an explanation